.dockerignore

```diff
@@ -1,2 +1,4 @@
 docker-machine*
-.git
```

Comment:
I have to say I'm still torn on this.  It will blow up the size of our build context significantly.  Why not re-work the scripts to pass the git SHA to the compile container as an environment variable and use that instead?